test/Microsoft.Extensions.CommandLineUtils.Tests/CommandLineApplicationTests.cs

```diff
@@ -253,8 +253,8 @@ public void AllowNoThrowBehaviorOnUnexpectedArgument()
 
             // (does not throw)
             app.Execute("test", unexpectedArg);
-            Assert.Equal(1, testCmd.RemainingArguments.Count);
-            Assert.Equal(unexpectedArg, testCmd.RemainingArguments[0]);
+            var arg = Assert.Single(testCmd.RemainingArguments);
```

Comment:
Hi @mikeharder. Glad to still be causing problems for .Net, even if indirectly :)